rllib/utils/test_utils.py

```diff
@@ -181,8 +181,9 @@ def check(x, y, decimals=5, atol=None, rtol=None, false=False):
         else:
             assert x == y, \
                 "ERROR: x ({}) is not the same as y ({})!".format(x, y)
-    # String comparison.
-    elif hasattr(x, "dtype") and x.dtype == np.object:
+    # String/byte comparisons.
+    elif hasattr(x, "dtype") and \
+            (x.dtype == np.object or str(x.dtype) == "<U216"):
```

Comment:
is "<U216" unicode string type or something?
can you help add a comment for us mortals :)

Reply:
Sorry, this was from another PR. I forgot to merge with master before crating this PR. It's all resolved now and this PR has nicely shrunk to 79 added lines. :)
But yes, this should have a comment! It's basically indicating a bytestring type.

Reply:
The comment is there though:
```
    # String/byte comparisons.
    elif hasattr(x, "dtype") and \
            (x.dtype == np.object or str(x.dtype) == "<U216"):
        ...
```